### PR TITLE
ci(travis): update node versions

### PR DESCRIPTION
**Pull request by sherwinski**
This PR removes node v4/adds v12 in the travis config
[More info](https://nodejs.org/en/about/releases/) on node LTS

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,6 @@
   "license": "BSD-2-Clause",
   "devDependencies": {
     "mocha": "2.5.3",
-    "phantomjs": "1.9.20",
```
Comment (jayeb) [inline package.json]: What's the story here?

Comment (sherwinski) [reply package.json]: I just wanted to test it out and see if anything else broke. Do we actually use this anywhere?

Comment (jayeb) [reply package.json]: It's almost certainly used by Mocha when running unit tests, but if things aren't breaking perhaps it's included in there as a peer dependency?

Comment (sherwinski) [reply package.json]: How about a compromise: we keep phantomjs but upgrade to prevent strange testing failures?

Comment (jayeb) [reply package.json]: It's a deal 🤝.

**Pull request merged**